haiku-apps/heidi/heidi-1.0.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="Native IDE for Haiku"
+DESCRIPTION="Heidi is a useful IDE for Haiku where coders can edit, \
+build, run, and debug their code projects all in one window."
+HOMEPAGE="https://github.com/waddlesplash/Heidi/"
```

Comment:
SUggestion: Drop the trailing slash in [`HOMEPAGE`](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#homepage)
```diff
-HOMEPAGE="https://github.com/waddlesplash/Heidi/"
+HOMEPAGE="https://github.com/waddlesplash/Heidi"
```